ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/144_dsc_negative_no_dependant_op.robot

```diff
@@ -25,36 +25,104 @@ ${IS_NOT_PRESENT}                           1
 
 
 *** Test Cases ***
-Validate DSC and DSCI Created Without Service Mesh And Serverless Operators     #robocop:disable
+Validate DSC and DSCI Created Without Service Mesh Operator     #robocop:disable
```

Comment:
maybe I'd go with something more like  `Validate DSC And DSCI Report Erorrs At Creation Time When ServiceMesh Is Not Installed` 

Otherwise I understand that it is checking DSC/DSCI can be created without service mesh, stop

Reply:
done @bdattoma. Changed the name and added a bit more of info in the documentation